ui/app/components/distribution-bar.js

```diff
@@ -76,6 +76,9 @@ export default Component.extend(WindowResizable, {
     const { chart, _data, isNarrow } = this.getProperties('chart', '_data', 'isNarrow');
     const width = this.$('svg').width();
     const filteredData = _data.filter(d => d.value > 0);
+    filteredData.forEach((d, index) => {
```

Comment:
I read a thing about preference of `for ... of` over `forEach` loops recently and have been trying to use them more. I certainly like they way they read, though that's not necessarily true of the transpiled code.

Reply:
I always forget that JS has better for loops now.